pkg/kubectl/cmd/get/get.go

```diff
@@ -206,6 +207,13 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 	}
 
 	sortBy, err := cmd.Flags().GetString("sort-by")
+
+	extraCols, err := cmd.Flags().GetStringSlice("extra-columns")
```

Comment:
I think this should follow the current guideline where we specify `-o custom-columns=...` but I'll defer to @smarterclayton who was requesting that.

Reply:
Alternatively we can support both.